README.md

```diff
@@ -16,6 +16,12 @@ url : /users/unfollow/
 method : DELETE
 parameters = {"username" : "<username of the user to be unfollowed">}
 ```
+#### Fetch Feed Posts
+To see posts from followed users and streams
+```
+url : /users/feed/
```

Comment:
shouldn't it be users/"<int:pk>"/feed
each user will have unique feed right

Reply:
But each user will see his own feed,  so once he is logged in there is no need for a primary key/